### PR TITLE
[bump_branch]: only add the suites that exist we dont need the script for this

### PR DESCRIPTION
**Pull request by jessfraz**
same fix as https://github.com/docker/docker/pull/21036 but for the bump branch

### CODE REVIEW

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, thanks Jess!

File: hack/make/release-deb [hunk 1]
```diff
@@ -73,7 +73,10 @@ TreeDefault {
 };
 EOF
 
-for suite in $(exec contrib/reprepro/suites.sh); do
+for dir in contrib/builder/deb/${PACKAGE_ARCH}/*/; do
+	version="$(basename "$dir")"
```
Comment (thaJeztah) [inline hack/make/release-deb]: oh, hold on, is this good (nested quotes)?

Comment (tiborvass) [inline hack/make/release-deb]: also the case in https://github.com/docker/docker/pull/21038/files#diff-c6c6bb0d2e164817a23d98d07109b25aR100 so its fine not updating it in this pr

Comment (jfrazelle) [inline hack/make/release-deb]: It's done on a line below too so yeah it works in my shell, and for
uniformity we should use the same or change both

On Tuesday, March 8, 2016, Sebastiaan van Stijn <notifications@github.com>
wrote:

> In hack/make/release-deb
> <https://github.com/docker/docker/pull/21038#discussion_r55417687>:
>
> > @@ -73,7 +73,10 @@ TreeDefault {
> >  };
> >  EOF
> >
> > -for suite in $(exec contrib/reprepro/suites.sh); do
> > +for dir in contrib/builder/deb/${PACKAGE_ARCH}/*/; do
> > +	version="$(basename "$dir")"
>
> oh, hold on, is this good (nested quotes)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/21038/files#r55417687>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by tiborvass** [comment]
this is not impacting CI, let's be nice to the CI, it needs some worship

**Comment by tianon** [comment]
LGTM (https://github.com/docker/docker/pull/21036#issuecomment-193938453)

**Comment by thaJeztah** [comment]
Thanks @tianon, I was like "how does _that_ work" for a bit :smile:

**Pull request merged**